code/modules/mob/living/simple_animal/hostile/giant_spider.dm

```diff
@@ -318,6 +282,123 @@
 	busy = SPIDER_IDLE
 	stop_automated_movement = FALSE
 
+// Midwives are upgraded nurses. They can web quickly and are stronger than regular nurses, but they're a bit slower.
+/mob/living/simple_animal/hostile/poison/giant_spider/nurse/midwife
+	name = "midwife"
+	desc = "Furry and black, it makes you shudder to look at it. This one has scintillating green eyes."
+	icon_state = "midwife"
+	icon_living = "midwife"
+	icon_dead = "midwife_dead"
+	maxHealth = 80
+	health = 80
+	speed = 2
+	web_speed = 0.15 // Easily able to web
+	// Allows the spider to use spider comms
+	var/datum/action/innate/spider/comm/letmetalkpls
+
+/mob/living/simple_animal/hostile/poison/giant_spider/nurse/midwife/Initialize(mapload)
+	. = ..()
+	letmetalkpls = new
+	letmetalkpls.Grant(src)
+
+/mob/living/simple_animal/hostile/poison/giant_spider/nurse/midwife/Destroy()
+	QDEL_NULL(letmetalkpls)
+	return ..()
+
+// Hunters have a decent amount of poison and have decent general stats, making them offensive spiders. They're a bit squishier
+// than regular spiders, though
+/mob/living/simple_animal/hostile/poison/giant_spider/hunter
+	name = "hunter"
+	desc = "Furry and black, it makes you shudder to look at it. This one has sparkling purple eyes."
+	icon_state = "hunter"
+	icon_living = "hunter"
+	icon_dead = "hunter_dead"
+	maxHealth = 80
+	health = 80
+	melee_damage = 20
+	poison_per_bite = 5
+	move_to_delay = 3
+	speed = 0
+
+// vipers are the upgraded variant of the hunter, able to move quickly and inject a lot of venom.
+/mob/living/simple_animal/hostile/poison/giant_spider/hunter/viper
+	name = "viper"
+	desc = "Furry and black, it makes you shudder to look at it. This one has effervescent purple eyes."
+	icon_state = "viper"
+	icon_living = "viper"
+	icon_dead = "viper_dead"
+	maxHealth = 50
+	health = 50
+	melee_damage = 1
+	poison_per_bite = 12
```

Comment:
```suggestion
	maxHealth = 35
	health = 35
	melee_damage = 5
	poison_per_bite = 12
```

Lethal in as little as one bite with patience and thought, they deserve to be squishier too. 5 melee_damage means they actually finish a victim paralyzed by venom in a reasonable timeframe while still needing to rely almost entirely on venom in order to kill.

Reply:
I left these guys with standard venom, instead of spider venom. They still have one damage because of that. 
I'm not entirely sure how good of an idea it is, but I wanted these guys to still be pretty lethal.

Ideally I'd still like to keep them at this stat level, since their health means that getting hit will put some pretty decent damage slowdown on them, and giving them less health will make it even easier for them to just die to atmos changes without being able to move.